src/lib/components/viz/histogram/OutlierHistogram.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+    import { fly } from "svelte/transition";
+    import { guidGenerator } from "$lib/util/guid";
+    import { DATA_TYPE_COLORS } from "$lib/duckdb-data-types";
+    import HistogramBase from "./HistogramBase.svelte";
+
+    export let data;
+    export let width;
+    export let height = 30;
+  
+    let left = 60;
+    let right = 4;
+    let top = 4;
+
+    export let mean: number;
+    export let sd: number;
+    export let min: number;
+    export let max: number;
+
+    export let anchorBuffer = 8;
+
+    const histogramID = guidGenerator();
+  
+    $: effectiveWidth = Math.max(width - 8, 120);
+
+    // add count for histogram base
+    $: data = data.map(datum =>  ({
+     ...datum,
+     count: 1       
+    }))
+
+    const intervals = [["µ", mean]];
+    $: ["1σ", "2σ", "3σ"].forEach((label, i) => {
+        // push interval only if it can be displayed on the plot
+        if ((mean + (i+1)*sd) <= max) intervals.push([label, mean + (i+1)*sd])
+        if ((mean - (i+1)*sd) >= min) intervals.push(["-"+label, mean - (i+1)*sd])
+    })
+
+  </script>
+  
+  <HistogramBase
+    data={data}
+    {left}
+    {right}
+    {top}
+    buffer={0}
+    width={effectiveWidth}
+    separate={false}
+    height={height + anchorBuffer * 3}
+    bottom={anchorBuffer * 3}
+    fillColor={DATA_TYPE_COLORS["DOUBLE"].vizFillClass}
+    baselineStrokeColor={DATA_TYPE_COLORS["DOUBLE"].vizStrokeClass}
+>
+<svelte:fragment let:x let:y let:buffer>
+    <filter id="outline-{histogramID}">
+      <feMorphology
+        in="SourceAlpha"
+        result="DILATED"
+        operator="dilate"
+        radius="1"
+      />
+      <feFlood flood-color="white" flood-opacity="1" result="PINK" />
+      <feComposite in="PINK" in2="DILATED" operator="in" result="OUTLINE" />
+
+      <feMerge>
+        <feMergeNode in="OUTLINE" />
+        <feMergeNode in="SourceGraphic" />
+      </feMerge>
+    </filter>
+    <g class="textElements">
+        {#each intervals as [label, value] (label)}
+            <line
+                x1={x(value)}
```

Comment:
i'm seeing type errors for this use of value, can you update that?-- 
![image](https://user-images.githubusercontent.com/2380975/173110967-a85bf6f8-2c50-4fa2-af44-d15df160bc42.png)

Reply:
Interesting, VSCode is not throwing any error for me. 
`x1` has type `(property) svelte.JSX.SVGAttributes<SVGLineElement>.x1?: string | number` when I inspect it. The `OutlierHistogram` component is derived from the `NumericHistogram` component. Do you see a similar type error there as well?

Reply:
For me the missing type was on `value`, which could be of type `number | string`. Looking farther up the file, I think `intervals` need a tuple type. This resolved it for me:

`const intervals:[string, number][] = [["µ", mean]];` (line 32)

Adding that clears the missing types on all those instanes of `value`.

If your VScode is not highlighting those, ping me on slack or something -- I can share my VSCode config to make sure we're seeng the same stuff ! :-)